evil-collection-vdiff.el

```diff
@@ -32,26 +32,35 @@
 
 (defun evil-collection-vdiff-setup ()
   "Set up `evil' bindings for `vdiff-mode'."
-  (evil-define-minor-mode-key 'normal 'vdiff-mode
-    "]c" 'vdiff-next-hunk
-    "[c" 'vdiff-previous-hunk)
-
-  ;; define `do' (diff obtain) and `dp' (diff put) bindings
-  (evil-define-minor-mode-key 'operator 'vdiff-mode
-    "o" '(menu-item
-          ""
-          nil
-          :filter (lambda (&optional _)
-                    (when (memq evil-this-operator
-                                evil-collection-delete-operators)
-                      #'vdiff-receive-changes)))
-    "p" '(menu-item
-          ""
-          nil
-          :filter (lambda (&optional _)
-                    (when (memq evil-this-operator
-                                evil-collection-delete-operators)
-                      #'vdiff-send-changes)))))
+
+  (dolist (mode '(vdiff-mode vdiff-3way-mode))
+    (evil-define-minor-mode-key 'normal mode
+      "zc" 'vdiff-close-fold
+      "zM" 'vdiff-close-all-folds
+      "zo" 'vdiff-open-fold
+      "zR" 'vdiff-open-all-folds
+      "]c" 'vdiff-next-hunk
+      "[c" 'vdiff-previous-hunk)
+    (evil-define-minor-mode-key 'motion mode
```

Comment:
Just noticed this. Is there a reason this is in motion? I don't use vdiff so I'm not familiar with it.

Reply:
Is this operator or normal?

Reply:
i moved `go` and `gp` back to normal state.